examples/formik/js/async-validator/index.js

```diff
@@ -0,0 +1,84 @@
+// Helper styles for demo
+import './helper.css';
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
+      Basic{' '}
+      <a
+        href="https://github.com/jaredpalmer/formik"
+        target="_blank"
+        rel="noopener"
+      >
+        Formik
+      </a>{' '}
+      Demo
+      <Formik
+        initialValues={{ user: '' }}
+        onSubmit={(values, { setSubmitting }) => {
+          setTimeout(() => {
+            alert(JSON.stringify(values, null, 2));
+            setSubmitting(false);
+          }, 500);
+        }}
+        validate={values => formValidation.validateForm(values)}
+      >
+        {props => {
+          const {
+            values,
+            touched,
+            errors,
+            dirty,
+            isSubmitting,
+            handleChange,
+            handleBlur,
+            handleSubmit,
+            handleReset,
+          } = props;
+          return (
+            <form onSubmit={handleSubmit}>
+              <label htmlFor="user" style={{ display: 'block' }}>
```

Comment:
Not neede style isn't it? We have it in `helper.css` file

Reply:
Same thing formik examples stuff